### DISCUSSION TITLE
Sporadic Inconsistent Results in PnP Search Result webpart

### DISCUSSION THREAD

**Discussion by pratibha08thakur** [microsoft-search/pnp-modern-search#3099] [Q&A] [answered]
**Inconsistent People Search results in PnP Search Modern webpart**
Sometimes the search result count in PnP search webpart for People search changes based on page refresh and user logged in

**Description of use case**
We have used PnP Modern Search webpart for the People search in our organisation but sometimes users reports they can't find certain users (any random user) in the page. Also, they see the different result count for the webpart. This issue occurs sporadically and show different results when page is refreshed or for different user.

**Examples**:

Showing 2040 result count(actual count) for People search webpart (PnP) and on refresh of page it changes to 2019 and again on refresh it changes back to 2040. For one user this count is 2040 and at the same time for different user it is 2019.
This happens sporadically not always. Issue get resolved automatically after some hours.

**Additional things to consider**
The webpart is used in different pages but same issue happens everywhere . Tried to connect with Microsoft as this could be MS search issue but they don't provide support for open source code (PnP). Also, the issue only happens with PnP search result webpart page not MS search as same user who is missing from PnP page is visible in MS search.

**Comment by kasperbolarsen** [comment, marked as answer]
as per https://github.com/microsoft-search/pnp-modern-search/issues/2527 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/c1f89004-09cf-48b5-be51-8e8227429703)

Perhaps you could try to check if you can see the same behavior in the API?
